embulk-output-sqlserver/src/test/resources/sqlserver/yml/test-jtds.yml

```diff
@@ -0,0 +1,43 @@
+in:
+  type: file
+  path_prefix: '/sqlserver/data/test1/test1.csv'
+  parser:
+    charset: UTF-8
+    newline: CRLF
+    type: csv
+    delimiter: ','
+    quote: ''
+    columns:
+    - {name: ID, type: string}
+    - {name: TINYINT_ITEM, type: long}
+    - {name: SMALLINT_ITEM, type: long}
+    - {name: INT_ITEM, type: long}
+    - {name: BIGINT_ITEM, type: long}
+    - {name: BIT_ITEM, type: boolean}
+    - {name: DECIMAL_ITEM, type: string}
+    - {name: NUMERIC_ITEM, type: string}
+    - {name: SMALLMONEY_ITEM, type: string}
+    - {name: MONEY_ITEM, type: string}
+    - {name: REAL_ITEM, type: double}
+    - {name: FLOAT_ITEM, type: double}
+    - {name: CHAR_ITEM, type: string}
+    - {name: VARCHAR_ITEM, type: string}
+    - {name: TEXT_ITEM, type: string}
+    - {name: NCHAR_ITEM, type: string}
+    - {name: NVARCHAR_ITEM, type: string}
+    - {name: NTEXT_ITEM, type: string}
+    - {name: DATE_ITEM, type: timestamp, format: '%Y/%m/%d'}
+    - {name: DATETIME_ITEM, type: timestamp, format: '%Y/%m/%d %H:%M:%S.%N'}
+    - {name: DATETIME2_ITEM, type: timestamp, format: '%Y/%m/%d %H:%M:%S.%N'}
+    - {name: DATETIME2_2_ITEM, type: timestamp, format: '%Y/%m/%d %H:%M:%S.%N'}
+    - {name: SMALLDATETIME_ITEM, type: timestamp, format: '%Y/%m/%d %H:%M:%S'}
+    - {name: TIME_ITEM, type: timestamp, format: '%H:%M:%S.%N'}
+    - {name: TIME_2_ITEM, type: timestamp, format: '%H:%M:%S.%N'}
+out:
+    type: sqlserver
+    host: localhost
+    database: TESTDB
+    user: TEST_USER
+    password: test_pw
+    table: TEST1
+    mode: replace
```

Comment:
If `testJtds` is a copy of `testNative`, `mode` should be `insert_direct` as `test-native.yml`.

Even so `testJtds` will fail because jTDS and Microsoft JDBC Driver will return different values.
For example, MS Driver will return Short values for TINYINT and SMALLINT, but jTDS Driver will return Integer values.
And MS Driver will return String values for TEXT and NTEXT, but jTDS Driver will return java.sql.Clob values.
And jTDS Driver will return String values for some of date/time types.